src/components/collections-grid/shop-by-room.js

```diff
@@ -34,15 +40,15 @@ export const ShopByRoom = ({ gallery, header, subheader, btnText }) => {
       </div>
       <div className={sty.buttonArea}>
         <div className={sty.horizontalLine}></div>
-        <button className={sty.shopButton}>
-          <span className={sty.btnContent}>
+        <div>
```

Comment:
Do we need this wrapping div or can we kill it?

Reply:
Was there to prevent the button from shrinking but just figured out what was causing it so removed the wrappers now!